packages/stark-ui/src/modules/date-picker/components/date-picker.component.ts

```diff
@@ -58,25 +106,76 @@ export class StarkDatePickerComponent extends AbstractStarkUiComponent implement
 	 * Whether the datepicker is disabled
 	 */
 	@Input()
-	public isDisabled?: boolean;
+	public get disabled(): boolean {
+		return this._disabled;
+	}
+	public set disabled(value: boolean) {
+		const newValue: boolean = coerceBooleanProperty(value);
+
+		if (this._disabled !== newValue) {
+			this._disabled = newValue;
+		}
+	}
+	private _disabled: boolean = false;
 
 	/**
 	 * Label to be displayed in the datepicker
 	 */
 	@Input()
 	public label: string;
 
+	/**
+	 * Timestamp Mask Configuration to apply on the date-picker.
+	 * If `true` is passed, the default mask config is applied:
+	 * ```
+	 * { format: "DD-MM-YYYY" }
+	 * ```
+	 */
+	@Input()
+	public get maskConfig(): StarkDatePickerMaskConfig {
+		return this._maskConfig;
+	}
+	public set maskConfig(config: StarkDatePickerMaskConfig) {
+		if (config === true) {
+			this._maskConfig = DEFAULT_DATE_MASK_CONFIG;
+		} else if (config) {
+			this._maskConfig = config;
+		} else {
+			this._maskConfig = undefined;
+		}
+	}
+
+	/**
+	 * @ignore
+	 * @internal
+	 */
+	private _maskConfig: StarkDatePickerMaskConfig = undefined;
```

Comment:
I would refactor this to:
```ts
@Input()
public set maskConfig(value: StarkTimestampMaskConfig | "default") {
	this._maskConfig = value === "default" ? DEFAULT_DATE_MASK_CONFIG : value;
}

/**
 * @ignore
 * @internal
 */
private _maskConfig: StarkTimestampMaskConfig;
```

Because in the class you can request `this._maskConfig` so the getter is not required. 
And if people do not want a mask they can just not set `maskConfig` instead of setting it to false.